clang/lib/Parse/ParseExpr.cpp

```diff
@@ -822,6 +822,7 @@ bool Parser::isRevertibleTypeTrait(const IdentifierInfo *II,
     REVERTIBLE_TYPE_TRAIT(__is_unbounded_array);
     REVERTIBLE_TYPE_TRAIT(__is_union);
     REVERTIBLE_TYPE_TRAIT(__is_unsigned);
+    REVERTIBLE_TYPE_TRAIT(__is_virtual_base_of);
```

Comment:
@cor3ntin Am I correctly handling this in the revertible type traits mechanism?

Reply:
We should not add revertible type traits for new traits. libstdc++ can use an other name if they need to create a struct

Reply:
Removed.